frontend/src/components/common/DatePicker.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <v-dialog
+    v-model="datePickerActive"
+    width="290px"
+  >
+    <template #activator="{ props: activatorProps }">
+      <v-text-field
+        id="datum"
+        v-model="textFieldDate"
+        variant="underlined"
+        validate-on="input"
+        :hint="displayFormat"
+        :disabled="disabled"
+        :required="required"
+        density="compact"
+        :rules="[
+            (toCheck) => validateTextDate(toCheck),
+          ]"
+        @blur="blur"
```

Comment:
Hier wurde vor kurzem in ISI ein Fix gemacht der das Event abfeuert wenn aus dem Textfeld gesprungen wird oder wenn ein Element über den Datepicker ausgewählt wird.